### PR TITLE
Improving ci test

### PR DESCRIPTION
**Pull request by AndreasArvidsson**

### CODE REVIEW

File: src/scripts/runTestsCI.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    if (error) {
+        console.log("error: ", error);
+    }
+
+    console.log("finished installing dependency extensions\n");
+}
+
+export async function launchVscodeAndRunTests() {
+    try {
+        const outDirPath = path.join(__dirname, "../../out");
```
Comment (pokey) [inline src/scripts/runTestsCI.ts:49]: This should point to root I believe

Comment (AndreasArvidsson) [reply src/scripts/runTestsCI.ts]: setting `extensionDevelopmentPath` to root it didn't help with the  command not found errors

Comment (pokey) [reply src/scripts/runTestsCI.ts]: Ok I guess either way you need to do that tho. It should point to the directory containing your package.json

Comment (pokey) [reply src/scripts/runTestsCI.ts]: Maybe put a log statement in extension activation to be sure it's loading?

Comment (AndreasArvidsson) [reply src/scripts/runTestsCI.ts]: I added a large debug print to the activate function and that is never run.

Comment (AndreasArvidsson) [reply src/scripts/runTestsCI.ts]: I've found the problem. In package.json I had specified that my extension depends on the command server. I didn't install the command server  in the test environment so the extension just silently failed to activate. No logs, no nothing the extension just would not activate. . Even worst when I used `vscode.extensions.all` and looked after my extension it actually had the boolean `isActive` set to true.

**Pull request merged**